lsp-ui-imenu.el

```diff
@@ -58,12 +59,27 @@
 (defcustom lsp-ui-imenu-colors '("deep sky blue" "green3")
   "Color list to cycle through for entry groups."
   :type '(repeat color)
-  :group 'lsp-ui-menu)
+  :group 'lsp-ui-imenu)
 
 (defcustom lsp-ui-imenu-window-width 0
   "When not 0, don't fit window to buffer and use value as window-width."
   :type 'number
-  :group 'lsp-ui-menu)
+  :group 'lsp-ui-imenu)
+
+(defcustom lsp-ui-imenu-auto-refresh t
+  "Automatically refresh imenu when certain conditions meet."
+  :type 'boolean
+  :group 'lsp-ui-imenu)
+
+(defcustom lsp-ui-imenu-refresh-delay 1.0
```

Comment:
If I understand correctly, this delay is used only when `lsp-ui-imenu-auto-refresh` is non-nil ?
If so, can you rename the variable to `lsp-ui-imenu-auto-refresh-delay`

Reply:
Changed! Thanks!